backend/app/core/container.py

```
@@ -37,11 +48,18 @@
     @property
-    def item_service(self) -> ItemService:
-        """Get item service instance."""
-        if self._item_service is None:
-            self._item_service = ItemService(self.session)
-        return self._item_service
+    def habit_service(self) -> HabitService:
+        """Get habit service instance."""
+        if self._habit_service is None:
+            self._habit_service = HabitService(self.session)
+        return self._habit_service
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)